kinopoisk/movie/sources.py

```diff
@@ -204,10 +204,16 @@ class MovieMainPage(KinopoiskPage):
         'rating': './/span[@class="rating_ball"]/text()',
         'votes': './/div[@id="block_rating"]//div[@class="div1"]//span[@class="ratingCount"]/text()',
         'imdb': './/div[@id="block_rating"]//div[@class="block_2"]//div[last()]/text()',
+        'imdb2': './/div[@id="block_rating"]//div[@class="block_2"]//div[last()-1]/text()',
```

Comment:
Could you please try `'.//div[@id="block_rating"]//div[@class="block_2"]//div[position()>last()-1]/text()'` for `imdb` index? I would like to keep as less xpath as possible.

Reply:
It doesn't work - https://travis-ci.org/ramusus/kinopoiskpy/jobs/657903683?utm_medium=notification&utm_source=github_status